src/libs/actions/App.js

```diff
@@ -63,6 +63,18 @@ function getNonOptimisticPolicyIDs(policies) {
         .value();
 }
 
+/**
+ * @param {Array} policies
+ * @return {Object} map of policy id to lastUpdated
+ */
+function getNonOptimisticPolicyIDToLastModifiedMap(policies) {
+    return _.chain(policies)
+        .reject((policy) => lodashGet(policy, 'pendingAction', null) === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD)
```

Comment:
NAB, default value can be empty string

Reply:
Really doesn't matter because we're comparing it to `CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD` and that's it. But sure!